package.json

```diff
@@ -27,7 +27,7 @@
     "chai": "^3.5.0",
     "css-loader": "^0.9.1",
     "except": "^0.1.3",
-    "extract-text-webpack-plugin": "^0.3.8",
+    "extract-text-webpack-plugin": "^3.0.0",
     "faker": "^2.1.2",
     "file-loader": "^0.8.1",
```

Comment:
Could you try updating file-loader?  Right now it causes a deprecation warning due to how it uses the loader-utils library.

> (node:27972) DeprecationWarning: loaderUtils.parseQuery() received a non-string value which can be problematic, see https://github.com/webpack/loader-utils/issues/56
parseQuery() will be replaced with getOptions() in the next major version of loader-utils.